metadata-ingestion/src/datahub/ingestion/source/metabase.py

```diff
@@ -244,7 +244,9 @@ def construct_dashboard_from_api_data(
         chart_urns = []
         cards_data = dashboard_details.get("ordered_cards", "{}")
         for card_info in cards_data:
-            chart_urn = builder.make_chart_urn(self.platform, card_info.get("id", ""))
+            chart_urn = builder.make_chart_urn(
+                self.platform, card_info.get("card_id", "")
+            )
```

Comment:
Not sure if this has changed in the Metabase API or if it has always been broken but `card_id` is the correct "foreign key" to `id` in the api/card enpoint.